apps/client/ui/InputText/InputText.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from 'react'
+import clsx from 'clsx'
+
+export interface InputProps {
+  size?: 'small' | 'medium' | 'large'
+  placeholder?: string
+  onClick?: () => void
+}
+
+export const getSizeClasses = (size: 'small' | 'medium' | 'large'): string => {
+  switch (size) {
+    case 'small': {
+      return 'w-48'
+    }
+    case 'large': {
+      return 'w-80'
+    }
+    default: {
+      return 'w-64'
+    }
+  }
+}
+
+export const BASE_INPUT_CLASSES =
```

Comment:
Maybe one more thing - you can think if it is worth to abstract these styles to use it in both inputs - maybe it would be redundant 😄  what we should abstract is the business responsibility and even if some code is the same it should be separated because the things are not related to each other. you might want to modify one because of some business reason and not the other one. let's consider if it is such case 🤓

Reply:
topic for tomorrow 📍